docs/providers/aws/guide/serverless.yml.md

```diff
@@ -54,6 +54,7 @@ provider:
       key2: value2
   deploymentPrefix: serverless # The S3 prefix under which deployed artifacts should be stored. Default is serverless
   role: arn:aws:iam::XXXXXX:role/role # Overwrite the default IAM role which is used for all functions
+  permissionsBoundary: arn:aws:iam::XXXXXX:policy/policy # ARN of an Permissions Boundary for the role.
```

Comment:
Maybe let's rename it to `rolePermissionsBoundary` (to correlate it more with `role` setting, as otherwise we also have `cfnRole`, `provider.logs.restApi.role` etc, even though in other cases we never construct the role)

Reply:
:+1: Done